playbook/roles/upcloud-servers/defaults/main.yml

```diff
@@ -0,0 +1,70 @@
+# These variables are used to give hostnames for the new machines
+# upcloud_server_hostname_base_domain: client-name.example.com
+# upcloud_project_name: Client-Name
+
+# This is a list of allowed ssh-keys for the root user after creating the new servers
+# upcloud_server_admin_ssh_keys:
+#  - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCzZ...
```

Comment:
I added automatic initial key population for the root user from the wunder key server.